.travis.yml

```diff
@@ -1,3 +1,46 @@
 language: ruby
 rvm:
- - 2.5.3
\ No newline at end of file
+  - 2.5.3
```

Comment:
Bcourses page 4 setup has 2.3.1 here, probably doesn't matter that we're using this version just wanted to make sure this was intentional